lib/vagrant-openshift/helper/command_helper.rb

```diff
@@ -160,7 +160,9 @@ def repo_checkout_bash_command(repo, url)
         git fetch origin
         git fetch --prune origin +refs/tags/*:refs/tags/*
         git reset --merge
-        git checkout #{branch}
+        if ! git checkout #{branch}; then
+          git checkout -b #{branch} origin/#{branch}
```

Comment:
@stevekuznetsov, actually, your last comment does make more sense.  I can just fix lib/vagrant-openshift/constants.rb.  I'll set the upstream to `https://github.com/openshift/openshift-ansible.git`.  That definitely makes things work and I'll see if that's all we need for now.  I suspect we'll be working on a master branch soon enough and won't even actually need this patch.